tests/files/gce_ubuntu18-flannel-aio.yml

```diff
@@ -2,11 +2,12 @@
 cloud_image_family: ubuntu-1804-lts
 cloud_region: us-central1-a
 cloud_machine_type: "n1-standard-1"
-mode: aio
+mode: ha
```

Comment:
aio mode should work. if it doesn't let's fix it.

Reply:
yeah, it doesn't work. need to fix.

Reply:
we just have to 'untaint' nodes that are both in master and node group

Reply:
> we just have to 'untaint' nodes that are both in master and node group

Should already happen: https://github.com/kubernetes-incubator/kubespray/pull/3466

Reply:
```
- name: set fact
  set_fact:
     masters_nodes: "{{groups['kube-master'] | intersect(groups['kube-nodes'])}}"

- name: untaint master+node hosts
  command: kubectl taint nodes {{masters_nodes|join(' ') }} node-role.kubernetes.io/master-
```

Reply:
> ```
> - name: set fact
>   set_fact:
>      masters_nodes: "{{groups['kube-master'] | intersect(groups['kube-nodes'])}}"
> 
> - name: untaint master+node hosts
>   command: kubectl taint nodes {{masters_nodes|join(' ') }} node-role.kubernetes.io/master-
> ```

Oh i see that #3466 only happens on kubeadm deployments. Sorry

Reply:
so with #3466 it should already work aio, could you try again @riverzhang

Reply:
@ant31  it still doesn't work.  i tested it.

Reply:
` Warning  FailedScheduling  112s (x34 over 6m54s)  default-scheduler  0/1 nodes are available: 1 Insufficient cpu.`
The cpu is too small to be scheduled.

Reply:
Are all resources requierements correctly set?